docs/manual/troubleshooting.tex

```diff
@@ -1062,6 +1062,70 @@ \subsection{Code style\label{code-style}}
 in the top-level directory that sets the property \code{maxparallel} to 4000.  If the error is still issued, reduce the number and
 try again.
 
+\section{Upgrading the stubparser module\label{upgrading-stubparser}}
```

Comment:
I think these instructions are better in typetools/stubparser.
Also, instead of committing new jar files, let's fix #1466 and properly integrate this project.